README.md

```diff
@@ -7,208 +7,194 @@
-<a href="https://opencollective.com/goby/backer/20/website" target="_blank"><img src="https://opencollective.com/goby/backer/20/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/21/website" target="_blank"><img src="https://opencollective.com/goby/backer/21/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/22/website" target="_blank"><img src="https://opencollective.com/goby/backer/22/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/23/website" target="_blank"><img src="https://opencollective.com/goby/backer/23/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/24/website" target="_blank"><img src="https://opencollective.com/goby/backer/24/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/25/website" target="_blank"><img src="https://opencollective.com/goby/backer/25/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/26/website" target="_blank"><img src="https://opencollective.com/goby/backer/26/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/27/website" target="_blank"><img src="https://opencollective.com/goby/backer/27/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/28/website" target="_blank"><img src="https://opencollective.com/goby/backer/28/avatar.svg"></a>
-<a href="https://opencollective.com/goby/backer/29/website" target="_blank"><img src="https://opencollective.com/goby/backer/29/avatar.svg"></a>
+Goby has several aspects: language specification, design of compiler and vm, implementation (just one for now), library, and the whole of them. 
 
+----------
 
-### Sponsors
+**Language**: Class-based, straignt-ahead object-oriented script language. Syntax is influenced by Ruby language (and by Go a bit), but has been **reduced and simplified** (and slightly modified) to keep Goby VM simple and concise. Some aspects of Ruby such as meta-programming (known as 'magic'), special variables with `$`, or like that, have been dropped. 
 
-Become a sponsor and get your logo on our README on Github with a link to your site. [[Become a sponsor](https://opencollective.com/goby#sponsor)]
+**Class**: Single inheritance, and 'module' is supported for mixin. Goby has internally two kinds of class: native class and standard class. **Native class** (or builtin class) provides fundamental methods. Most common methods such as `#puts` are in `Class` class. `Object` class is just a root class for consistency and holds no methods. **Standard class** (or standard library) can be loaded via `require` and provides additional methods. Standard classes are often split internal Go code and external Goby code in order to make implementation easier. Both kind of classes are transparent to Goby developers and can be overridden by child classes. Goby's Class hierarchy is very simple: `Object::Class::(anything like that)`.
```

Comment:
I'm not sure what do you mean about the Class hierarchy. All constants is under `Object` namespace, but not necessarily under `Class` namespace.